meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb

```diff
@@ -25,8 +26,10 @@ inherit deploy
 do_configure[noexec] = "1"
 
 do_compile() {
-	sed -e '/@@INCLUDE_COMMON@@/ {' -e 'r ${S}/boot-common.cmd.in' -e 'd' -e '}' \
+	sed -e '/@@INCLUDE_COMMON_IMX@@/ {' -e 'r ${S}/boot-common-imx.cmd.in' -e 'd' -e '}' \
 			"${S}/boot.cmd" > boot.cmd.in
```

Comment:
This is fine for now, but later on we can probably change this to INCLUDE_COMMON_BSP and have the boot-common-bsp.cmd.in file to be provided under a bsp specific folder (e.g. imx here), as then we would have a more generic way of adding stuff that is specific to the bsp.